_data/library/scalalibs.yml

```diff
@@ -325,3 +357,7 @@
       url: https://github.com/jokade/sbt-node
       desc: A Sbt plugin for integration with npm / node.js based build tools.
       dep: 'addSbtPlugin("de.surfice" % "sbt-node" % "0.0.2")'
+    - name: sbt-npm-package
+      url: https://github.com/davenverse/sbt-npm-package
+      desc: For publishing Scala.js packages to npm.
+      dep: 'addSbtPlugin("io.chrisdavenport" %% "sbt-npm-package" % "0.1.0")'
```

Comment:
I could not find any kind of documentation for this project at all. As is I don't think it's ready to be listed here, since it's impossible to use it if you're not already a contributor who knows what this is supposed to do.

Reply:
You're right. I removed it.